### PR TITLE
add doc for "Precision Math"

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: sql/precision-math.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+| `sql_mode` 的值 | 结果 |
+| :--- | :--- |
+| '' | 没有 warning，没有 error，i 被设为 NULL |
+| strict | 没有 warning，没有 error，i 被设为 NULL |
+| `ERROR_FOR_DIVISION_BY_ZERO` | 有 warning，没有 error，i 被设为 NULL |
+| strict, `ERROR_FOR_DIVISION_BY_ZERO` | 有 error，插入失败 |
+
+
+## <span id="round">数值修约</span>
```
Comment (andelf) [inline sql/precision-math.md]: do not use html in markdown

File: sql/precision-math.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+TiDB > SELECT ROUND(2.5), ROUND(25E-1);
++------------+--------------+
+| ROUND(2.5) | ROUND(25E-1) |
++------------+--------------+
+|          3 |            3 |
++------------+--------------+
+1 row in set (0.00 sec)
+```
+
+向一个 `DECIMAL` 或者整数类型列插入数据时，round 的规则将采用 [round half away from zer](https://en.wikipedia.org/wiki/Rounding\#Round_half_away_from_zero) 的方式：
```
Comment (andelf) [inline sql/precision-math.md]: typo

File: sql/precision-math.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+TiDB > SELECT ROUND(2.5), ROUND(25E-1);
++------------+--------------+
+| ROUND(2.5) | ROUND(25E-1) |
++------------+--------------+
+|          3 |            3 |
++------------+--------------+
+1 row in set (0.00 sec)
+```
+
+向一个 `DECIMAL` 或者整数类型列插入数据时，round 的规则将采用 [round half away from zero](https://en.wikipedia.org/wiki/Rounding\#Round_half_away_from_zero) 的方式：
```
Comment (datahoecn) [inline sql/precision-math.md]: if .. or not ()  ->  round

**Pull request merged**